aws/templates/commonApplication.ftl

```diff
@@ -253,193 +253,188 @@
 
     [#local containers = [] ]
 
-    [#list (configuration.Containers!{})?values as container]
-        [#if container?is_hash]
-
-            [#local portMappings = [] ]
-            [#list (container.Ports!{})?values as port]
-                [#if port?is_hash]
-                    [#local targetLoadBalancer = {} ]
-                    [#local targetTierId = (port.LB.Tier)!port.ELB?has_content?then("elb", "") ]
-                    [#local targetComponentId = port.LB.Component!port.ELB?has_content?then(port.ELB, "") ]
-                    [#local targetGroup = (port.LB.TargetGroup)!""]
-                    [#local targetPath = (port.LB.Path)!""]
-                    [#local targetType = port.ELB?has_content?then("elb", "alb")]
-                    
-                    [#if targetTierId?has_content && targetComponentId?has_content]
-                        [#-- Work out which occurrence to use --]
-                        [#local targetComponent =
-                            getComponent(
-                                targetTierId,
-                                targetComponentId,
-                                targetType)]
-                        [#local instanceAndVersionMatch = {}]
-                        [#local instanceMatch = {}]
-                        [#if targetComponent?has_content]
-                            [#list getOccurrences(getTier(targetTierId), targetComponent) as targetOccurrence]
-                                [#if core.Instance.Id == targetOccurrence.Core.Instance.Id]
-                                    [#if core.Version.Id == targetOccurrence.Core.Version.Id]
-                                        [#local instanceAndVersionMatch = targetOccurrence]
-                                    [#else]
-                                        [#if (core.Version.Id?has_content) &&
-                                                (!(targetOccurrence.Core.Version.Id?has_content))]
-                                            [#local instanceMatch = targetOccurrence]
-                                        [/#if]
-                                    [/#if]
+    [#list configuration.Containers?values as container]
```

Comment:
With resources being decentralised more and more should the container configuration be moved into either its own files or into the ECS configuration?

Reply:
Originally this routine was shared by lambda but its now only used by ecs - so I think it could move. However the application_* templates are designed to be built into a switch statement so the current location is the most logical place at the moment.